### PR TITLE
add contour_expand and pixel_group in parrots

### PR DESCRIPTION
**Pull request by luopeichao**
Add pixel_group and contour_expand in parrots.

### CODE REVIEW

File: mmcv/ops/contour_expand.py [hunk 1]
```diff
@@ -33,6 +33,18 @@ def contour_expand(kernel_mask, internal_kernel_label, min_kernel_area,
     if isinstance(internal_kernel_label, np.ndarray):
         internal_kernel_label = torch.from_numpy(internal_kernel_label)
 
-    label = ext_module.contour_expand(kernel_mask, internal_kernel_label,
-                                      min_kernel_area, kernel_num)
+    if torch.__version__ == 'parrots':
+        if kernel_mask.shape[0] == 0 or internal_kernel_label.shape[0] == 0:
+            label = []
+        else:
```
Comment (zhouzaida) [inline mmcv/ops/contour_expand.py:39]: hi, can we move those lines to `ext_module.contour_expand`

File: mmcv/ops/pixel_group.py [hunk 2]
```diff
@@ -48,8 +48,29 @@ def pixel_group(score, mask, embedding, kernel_label, kernel_contour,
...
+        label = label[0]
+        list_index = kernel_region_num
+        output = []
+        for x in range(kernel_region_num):
+            output.append(
+                np.array(
+                    label[list_index:list_index + int(label[x])],
+                    dtype=np.float))
+            list_index = list_index + int(label[x])
+        pixel_assignment = output
```
Comment (zhouzaida) [inline mmcv/ops/pixel_group.py:70]:
```suggestion
        pixel_assignment = []
        for x in range(kernel_region_num):
            pixel_assignment.append(
                np.array(
                    label[list_index:list_index + int(label[x])],
                    dtype=np.float))
            list_index = list_index + int(label[x])
```

File: mmcv/ops/contour_expand.py [hunk 3]
```diff
@@ -33,6 +33,18 @@ def contour_expand(kernel_mask, internal_kernel_label, min_kernel_area,
...
+            label = ext_module.contour_expand(
+                kernel_mask,
+                internal_kernel_label,
+                min_kernel_area=min_kernel_area,
+                kernel_num=kernel_num)
+            label = label.tolist()
+        label = [np.array(m, dtype=np.int) for m in label]
```
Comment (ZwwWayne) [inline mmcv/ops/contour_expand.py:46]: Should we return ndarray or a tensor here? It does not seem to be matched with the docstring and the return type of label when using pytorch.

**Review by grimoire — APPROVED** [review]
LGTM

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**